packages/migrate/src/__tests__/DbPull/__snapshots__/sqlite.test.ts.snap

```diff
@@ -163,6 +163,36 @@ model AwesomeProfile {
 
 `;
 
+exports[`common/sqlite should succeed when --local-d1 and a single local Cloudflare D1 database exists 2`] = `
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["driverAdapters"]
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = "file:.wrangler/state/v3/d1/miniflare-D1DatabaseObject/5d11bcce386042472d19a6a4f58e40041ebc5932c972e1449cbf404f3e3c4a7a.sqlite"
```

Comment:
I'm not sure if that snapshot will be reliable with this value.

Reply:
The snapshots is reliable, as the `packages/migrate/src/__tests__/fixtures/cloudflare-d1-many-dbs/.wrangler/state/v3/d1/miniflare-D1DatabaseObject/5d11bcce386042472d19a6a4f58e40041ebc5932c972e1449cbf404f3e3c4a7a.sqlite` database has a fixed `5d11bcce386042472d19a6a4f58e40041ebc5932c972e1449cbf404f3e3c4a7a.sqlite` name :)

Reply:
Nice!